config/routes.rb

```diff
@@ -9,11 +9,11 @@
   end
 
   namespace :admin do
```

Comment:
I think you can set the admin constraint on the namespace instead of setting it on each of the resources.

Reply:
Ahh yes you can!